df.items.xml

```diff
@@ -561,7 +561,7 @@
 
             <vmethod name='getItemDescription'>
                 <pointer type-name='stl-string'/>
-                <int8_t name='mode'/>
+                <int8_t name='plurality' comment="0 = prickle berries [2], 1 = prickle berry, 2 = prickle berries"/>
```

Comment:
Looks good. At first I thought it was [`part_of_speech`](https://github.com/DFHack/df-structures/blob/2d31879f817731c4e2a1f17966900b67089ed32d/df.language.xml#L21), but it's definitely not the same order. I'm not seeing an existing enum like this, and defining one inline might not work since this is in a vmethod (but usually it would be a good idea). Anyway, unless you know of another place that uses 0/1/2 like this (I don't), this looks good to me. :+1: